content/manuals/engine/daemon/prometheus.md

```diff
@@ -41,15 +41,15 @@ Add the following configuration:
 
 ```json
 {
-  "metrics-addr": "127.0.0.1:9323"
+  "metrics-addr": "0.0.0.0:9323"
```

Comment:
Hm... wondering what's best here, because I think this is an unprotected port; not sure if we should document it as default example

cc @vvoland @akerouanton